Modules/_threadmodule.c

```diff
@@ -1022,6 +1022,13 @@ rlock_traverse(PyObject *self, visitproc visit, void *arg)
     return 0;
 }
 
+/*
+helper function used by rlock_locked and rlock_repr.
+*/
+static int
+rlock_locked_impl(rlockobject *self) {
```

Comment:
CPython formats functions like:

```suggestion
static int
rlock_locked_impl(rlockobject *self)
{
```